ddl/ddl_api.go

```diff
@@ -2594,6 +2595,7 @@ func (d *ddl) preSplitAndScatter(ctx sessionctx.Context, tbInfo *model.TableInfo
 }
 
 func (d *ddl) FlashbackCluster(ctx sessionctx.Context, flashbackTS uint64) error {
+	logutil.BgLogger().Info("[ddl] get flashback cluster job", zap.String("flashbackTS", oracle.GetTimeFromTS(flashbackTS).String()))
```

Comment:
Now, We have no guide on how to name a variable in the log. I check the Alibaba log guide. They use `_` as a joiner.

Reply:
I don't get your point. I think we have zap already, need have other joiner?

Reply:
```flashback_ts```  may be better than ```flashbackTs```.

Reply:
ok, maybe it need a lint tool to help us improve it.

Reply:
In the earlier version(probably around v3.0), we unified the log format, which was `flashbackTS`.

Reply:
😂😂